File: VB-Examples/20_Security/CheckDigitalSignature.vb
Imports System.Text
Imports Spire.Doc
Imports Spire.Doc.Documents
Imports Spire.Doc.Formatting
Imports System.IO

Namespace CheckDigitalSignature
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
				Dim hasDigitalSignature As Boolean = Document.HasDigitalSignature("..\..\..\..\..\..\Data\CheckDigitalSignature.docx")

				' Use a switch statement to determine the file format and update the fileFormat string accordingly
				If hasDigitalSignature Then
					MessageBox.Show("This Word document has digital signature")
				Else
					MessageBox.Show("This Word document has not digital signature")
				End If
		End Sub
	End Class
End Namespace

File: CS-Examples/20_Security/CheckDigitalSignature.cs
using System;
using System.Windows.Forms;
using System.Text;
using Spire.Doc;
using Spire.Doc.Documents;
using System.Drawing;
using Spire.Doc.Formatting;
using System.IO;

namespace CheckDigitalSignature
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
                bool hasDigitalSignature = Document.HasDigitalSignature(@"..\..\..\..\..\..\Data\CheckDigitalSignature.docx");

				// Use a switch statement to determine the file format and update the fileFormat string accordingly
				if(hasDigitalSignature)
                {
                    MessageBox.Show("This Word document has digital signature");
                }else
                {
                    MessageBox.Show("This Word document has not digital signature");
                }
        }
    }
}
